docs/source/blocks/settings.md

```diff
@@ -1,10 +1,10 @@
 ---
 myst:
   html_meta:
-    "description": "How to configure custom blocks"
-    "property=og:description": "How to configure custom blocks"
-    "property=og:title": "Blocks settings"
-    "keywords": "Volto, Plone, frontend, React, Block settings"
+    'description': 'How to configure custom blocks'
+    'property=og:description': 'How to configure custom blocks'
+    'property=og:title': 'Blocks settings'
+    'keywords': 'Volto, Plone, frontend, React, Block settings'
```

Comment:
Please revert all changes of `"` to `'`, and configure your editor not to make these changes.